### ISSUE TITLE
Accessibility: The interface is too small

### ISSUE THREAD

**Issue by Dekamir** [subhra74/xdm#62]
![Screenshot (373)](https://user-images.githubusercontent.com/28929300/77306351-99757880-6d08-11ea-9616-b1ea4ffe995a.png)

**The resolution is 2560x1080. Aspect ratio: 21:9 (or 64:27)**
My blind eyes can hardly read what's written there, even with my glasses.
_My screen is not big as it looks in the screenshot._

This wasn't a problem in the previous version.
I kindly request a scaling option for the application.

_Awesome software by the way. If you ever need a graphic designer, feel free to message._

**Comment by huyndnse63331** [comment]
indeed, you need to fix this @subhra74 , our eyes are being burned 🔥 🔥 🔥

**Comment by subhra74** [comment]
> ![Screenshot (373)](https://user-images.githubusercontent.com/28929300/77306351-99757880-6d08-11ea-9616-b1ea4ffe995a.png)
> 
> **The resolution is 2560x1080. Aspect ratio: 21:9 (or 64:27)**
> My blind eyes can hardly read what's written there, even with my glasses.
> _My screen is not big as it looks in the screenshot._
> 
> This wasn't a problem in the previous version.
> I kindly request a scaling option for the application.
> 
> _Awesome software by the way. If you ever need a graphic designer, feel free to message._

trying to find an work around, might need some input from you, as I do not have access to HD monitors for testing.

**Comment by subhra74** [comment]
Could you please generate the log using below method and post the entire log here.
https://github.com/subhra74/xdm/wiki/Generate-log-for-troubleshooting
Also please share the screen display settings:
![image](https://user-images.githubusercontent.com/8403090/77316682-27059800-6d0a-11ea-8ef4-f3438bd026e2.png)

**Comment by Dekamir** [comment]
_You can ask for detailed/tech savvy stuff, I can debug things for you easily._
**Resolution: 2560x1080, Scaling: %100**
This is the same screen resolution as your screenshot except mine is a bit wider. So still 1080p.

```
[ main ] loading...
[ main ] 11.0.6 10.0
[ main ] starting monitoring...
[ Thread-0 ] instance starting...
[ Thread-0 ] Init app
[ Thread-0 ] Loading fonts
[ Thread-0 ] Loading config...
[ Thread-0 ] Creating folders
en
[ Thread-0 ] Loading language en
[ Thread-0 ] Context initialized
[ Thread-0 ] instance started.
[ AWT-EventQueue-0 ] showing main window.
[ AWT-EventQueue-0 ] Dpi scale init: 1.0
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
icon type:hdpi
[ AWT-EventQueue-0 ] List changed
icon type:hdpi
icon type:hdpi
checking for app update
[ Thread-1 ] manifest download response: 200
7.2.10 7.2.10
```

**Comment by subhra74** [comment]
Thanks, I will contact you for testing display related issue fixes then.

For now I can see from the log that the dpi ratio (scaling) is 1. How does other applications look ( windows explorer/vlc etc). Can you please post a screenshot of them along with XDM. Also what happens if you increase the Scaling in your systems display settings. Does that fix the problem with XDM?

**Comment by Dekamir** [comment]
My computer was broken, I'll reply back to you ASAP as my installation completes.

**Comment by Dekamir** [comment]
Sorry for the hiatus. Mobo was broken.

Here's the comparison with Windows Explorer on Windows 7.
https://i.imgur.com/I6hXzJw.png

Although I'm kinda used to it, I still would love an option to get it big (because of my eye problem), or an overhaul on the user interface, which I can help from the ground up.

**Comment by subhra74** [comment]
Thanks a lot for your support, will provide an option for configurable scaling in next version

**Comment by KrasnayaPloshchad** [comment]
This problem could also make the interface uncomfortable on Windows touch screen (such as Surface).

**Comment by Dekamir** [comment]
> This problem could also make the interface uncomfortable on Windows touch screen (such as Surface).

How about this?
![XDM-Concept-R](https://user-images.githubusercontent.com/28929300/80270110-1015e500-86be-11ea-99d3-db092671c73f.png)

**Comment by subhra74** [comment]
Wow! that looks really nice. thanks a lot for the effort. XDM will look similar to this in future.

**Comment by validatedev** [comment]
@subhra74 the ui is incredibly small with my hidpi screen in linux, and zoom feature is completely broken.

**Comment by subhra74** [comment]
> @subhra74 the ui is incredibly small with my hidpi screen in linux, and zoom feature is completely broken.

Please explain with details information and screenshots

**Comment by fakhamatia** [comment]
HiDPI problem in Ubuntu 19.10
![Screenshot from 2020-07-21 18-13-27](https://user-images.githubusercontent.com/20030910/88062324-ee71b000-cb7d-11ea-9be7-06773a2936d2.png)

**Comment by validatedev** [comment]
@subhra74 
![little](https://user-images.githubusercontent.com/29679552/92415544-1a62e800-f162-11ea-8866-66dec148c082.png)

**Comment by joe733** [comment]
> Wow! that looks really nice. thanks a lot for the effort. XDM will look similar to this in future.

Hi, can you please point to where (branch) the UI development is going on?. Is it possible to let the OS / gtk / whatever native to render the UI for XDM so that it blends in with other applications?

**Comment by joe733** [comment]
> Wow! that looks really nice. thanks a lot for the effort. XDM will look similar to this in future.

Hi, can you please point to where (branch) the UI development is going on?. Is it possible to let the OS / gtk / whatever native to render the UI for XDM so that it blends in with other applications?

**Comment by Wiley2046** [comment]
> Thanks a lot for your support, will provide an option for configurable scaling in next version

Looking forward to your new version of the software.
